pyquil/tests/test_operator_estimation.py

```diff
@@ -1,21 +1,32 @@
+import functools
 import itertools
 import random
 from math import pi
-
-import numpy as np
-import functools
 from operator import mul
 
+import numpy as np
 import pytest
 
+from pyquil import Program, get_qc
 from pyquil.api import WavefunctionSimulator
+from pyquil.gates import *
 from pyquil.operator_estimation import ExperimentSetting, TomographyExperiment, to_json, read_json, \
-    _all_qubits_diagonal_in_tpb, group_experiments, ExperimentResult, measure_observables, \
-    _get_diagonalizing_basis, _max_tpb_overlap, group_experiments_greedy, \
-    _expt_settings_diagonal_in_tpb
+    group_experiments, ExperimentResult, measure_observables, SIC0, \
+    SIC1, SIC2, SIC3, plusX, minusX, plusY, minusY, plusZ, minusZ, \
+    _max_tpb_overlap, _max_weight_operator, _max_weight_state, \
+    TensorProductState, zeros_state
 from pyquil.paulis import sI, sX, sY, sZ, PauliSum, PauliTerm
-from pyquil import Program, get_qc
-from pyquil.gates import *
+
+
+def _generate_random_states(n_qubits, n_terms):
+    oneq_states = [SIC0, SIC1, SIC2, SIC3, plusX, minusX, plusY, minusY, plusZ, minusZ]
```

Comment:
Im flying blind here, so apologies if this doesn't make sense.

It seems like one might want to generate a random state from the Pauli eigenstates or from the SIC states but not from both.

Reply:
this is just for the tests, so it doesn't make perfect sense; but it _is_ something you can do (in code)